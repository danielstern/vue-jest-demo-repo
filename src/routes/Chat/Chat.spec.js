import Chat  from './Chat.vue';

describe('Chat Service', () => {

       

})