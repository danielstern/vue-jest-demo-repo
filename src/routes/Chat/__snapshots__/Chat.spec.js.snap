// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chat Service should match the snapshot 1`] = `
<div>
  <h1>
    


        Even Better Globoticket Customer Service Chat 

    
  </h1>
   
  <div
    class="card mb-3"
  >
    <div
      class="card-body"
    />
  </div>
   
  <div>
    <form
      class="form-inline"
    >
      <div
        class="form-group mb-2 mr-2"
      >
        <input
          class="form-control"
          placeholder="Your message"
          type="text"
        />
      </div>
       
      <button
        class="btn btn-primary mb-2"
        type="submit"
      >
        Send
      </button>
    </form>
  </div>
</div>
`;
